frontend/src/components/project.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import Image from 'next/image';
+import { Project } from '@/types';
+import 'pure-react-carousel/dist/react-carousel.es.css';
+import Link from 'next/link';
+import Toaster from '@/utils/toaster';
+import { Buildings } from '@phosphor-icons/react';
+import 'pure-react-carousel/dist/react-carousel.es.css';
+import { USER_PROFILE_PIC_URL } from '@/config/routes';
+import postHandler from '@/handlers/post_handler';
+import { SERVER_ERROR } from '@/config/errors';
+import { X } from '@phosphor-icons/react/dist/ssr';
+
+interface Props {
+  project: Project;
+  setProjects: React.Dispatch<React.SetStateAction<Project[]>>;
+}
+
+const ProjectComponent = ({ project, setProjects }: Props) => {
+  const removeFlag = () => {
+    const toaster = Toaster.startLoad('Removing Flag', project.id);
+    const URL = `/flags/projects/${project.id}`;
+    postHandler(URL, {})
+      .then(res => {
+        if (res.statusCode === 200) {
+          setProjects(prev => prev.filter(p => p.id != project.id));
+          Toaster.stopLoad(toaster, 'Flag Removed', 1);
+        } else {
+          if (res.data.message) Toaster.stopLoad(toaster, res.data.message, 0);
+          else {
+            Toaster.stopLoad(toaster, SERVER_ERROR, 0);
+          }
+        }
+      })
+      .catch(err => {
+        Toaster.stopLoad(toaster, SERVER_ERROR, 0);
+      });
+  };
+  return (
+    <div className="w-full">
+      <div className="w-full relative bg-white font-primary flex gap-1 rounded-lg rounded-b-none p-2 text-primary_black border-gray-300 border-b-0 border-[1px]">
+        <div className="h-full">
+          <Link href={`${`/explore/user/${project.user.username}`}`} className="rounded-full">
```

Comment:
project card should show details about the project. (title, description, created_at, etc)